dask/base.py

```diff
@@ -96,7 +97,7 @@ def compute(self, **kwargs):
     @classmethod
     def _get(cls, dsk, keys, get=None, **kwargs):
         get = get or _globals['get'] or cls._default_get
-        dsk2 = cls._optimize(dsk, keys, **kwargs)
+        dsk2 = cls._optimize(dict(dsk), keys, **kwargs)
```

Comment:
I haven't run into this during profiling.  I'm tempted to stick with the recognizable `dict` function until it becomes an issue.